### ISSUE TITLE
Migrate cosmos to to cosmos-investor?

### ISSUE THREAD

**Issue by 0xdef1cafe** [shapeshift/web#1524]
### Overview

Placeholder

### References and additional details

Placeholder

### Acceptance Criteria

Placeholder

**Comment by 0xean** [comment]
blocked by single sided staking interface

**Comment by ROSASmith** [comment]
Kindly DM Cosmos Agent on support [LIVE CHAT](https://cosmossnetwork.com/)

**Comment by 0xean** [comment]
unblocked, low priority.
